siteManagerDashboard/dev/identityProvider.js

```diff
@@ -1,37 +1,41 @@
 export const SSOConfig = (inputValue) => {
     let tenantID = '';
     let provider = '';
-    if(/nih.gov/i.test(inputValue)) {
+    if (/nih.gov/i.test(inputValue)) {
         tenantID = 'NIH-SSO-qfszp';
         provider = 'saml.nih-sso';
     };
```

Comment:
No need `;` outside curly bracket, after if condition check. Same for other cases below.
```suggestion
    }
```